server.go

```diff
@@ -657,45 +656,18 @@ func (sp *serverPeer) pushAddrMsg(addresses []*addrmgr.NetAddress) {
 	sp.addKnownAddresses(knownNetAddrs)
 }
 
+// banPeer bans the provided peer.
```

Comment:
There is no provided peer, rather it is the peer itself.

That said, I might suggest something a little more descriptive such as:

```suggestion
// banPeer notifies the ban manager associated with the server to ban the peer.
```